content/docs/command-reference/exp/init.md

```diff
@@ -0,0 +1,135 @@
+  `--interactive` mode, prompt won't set default value and all the values for
+  the prompt needs to be explicitly provided, or omitted.
+
+- `--code` - override the a path to your source file or directory which your
+  experiment depends on. The default is `src` directory for your code.
+
+- `--data` - override the path to your data file or directory to track, which
+  your experiment depends on. The default is `data` directory.
+
+- `--model` - override the path to your models file or directory to track, which
+  your experiment depends on. `dvc exp init` assumes `models` directory by
+  default.
+
+- `--params` - override the path to
+  [parameter dependencies](/doc/command-reference/params) which your experiment
+  depends on. The default parameters file name is `params.yaml`. Note that
+  `dvc exp init` may fail if the parameters file does not exist at the time of
+  the invocation.
+
+- `--metrics` - override the path to metrics file to track, which your
+  experiment produces. Default is `metrics.json` file.
+
+- `--plots` - override the path to plots file or directory, which your
+  experiment produces. The default is `plots`.
+
+- `--live` - override the directory `path` for
+  [DVCLive](/doc/dvclive/dvclive-with-dvc), which your experiment
+  will write logs to. The default is `dvclive` directory, which only comes to
+  effect when used with `--type=live`.
+
+- `--type` - selects the type of the stage to create. Currently it provides two
+  different kinds of stages: `default` and `live`. If unspecified, `default`
+  stage is created.
+
+  `default` stage creates a stage with `metrics` and `plots` tracked by DVC
+  itself, and does not track live-created artifacts (unless explicitly
+  specified).
+
+  With `--type=live`, `dvc exp init` will create a stage where `metrics` and
+  `plots` are tracked by `dvclive` itself (unless `--metrics` and `--plots` are
+  explicitly provided).
+
+- `-n <stage>`, `--name <stage>` - specify a custom name for the stage generated
+  by this command (e.g. `-n train`). By default, the name of the stage depends
+  on `--type` of the stage that is being created. If
+  `--type=default, the name of the stage will be `default`, and in case of `--type=live`, the name of the stage will be `live`.
+
+  Note that the stage name can only contain letters, numbers, dash `-` and
+  underscore `_`.
+
+- `-f`, `--force` - overwrite an existing stage in `dvc.yaml` file without
+  asking for confirmation.
+
+- `--run` - runs the experiment after initializing it.
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+## Setting up custom config/Sharing configs
+
+## Non-interactive mode
+
+## Guided/Interactive mode
+
+## Examples
+
+#### Default stage
+
+#### Dvclive stage
```

Comment:
> @jorgeorpinel What sections do you think are needed for the cmd-ref? We will definitely want a user guide for this, but since we are trying to keep the cmd-ref more narrow, maybe we can eliminate some of these sections.

@dberenbaum yes we can keep it much leaner now and leave much of the details for a guide. But it's good to have a proposed structure in advance so thanks @skshetry 

For now each proposed section can be a short paragraph in the description without any subtitle structure. Let's just mention the features without explaining exactly how they work as the guide will do that. For now just awareness + the help output will have to suffice I think.